src/gui/application.cpp

```diff
@@ -396,21 +396,8 @@ Application::~Application()
     AccountManager::instance()->shutdown();
 }
 
-void Application::slotAccountStateRemoved(AccountStatePtr accountState) const
+void Application::slotAccountStateRemoved() const
 {
-    if (_gui) {
```

Comment:
@fmoc @erikjv @dschmidt @dragotin as far as I can tell we don't need to disconnect those as once the emitting object is gone they can no longer be triggered....